### DISCUSSION TITLE
Sync icon badge " ! " what does it mean?

### DISCUSSION THREAD

**Discussion by quicksite** [sienori/Tab-Session-Manager#813] [Q&A] [answered]
Screenshot:  https://i.imgur.com/Qc99AMs.png
<img width="780" alt="Screen Shot 2021-06-18 at 7 12 04 PM" src="https://user-images.githubusercontent.com/128756/122628439-d2ff7580-d06a-11eb-8bca-0520cfcc7052.png">

What does the orange exclamation point indicate re a Sync problem?
Is there any corrective action to take?

NOTE:  I've read in extension Settings that this is a BETA feature:
<img width="755" alt="Screen Shot 2021-06-18 at 7 27 34 PM" src="https://user-images.githubusercontent.com/128756/122628547-7a7ca800-d06b-11eb-9489-6c55f21345a1.png">

**Comment by sienori** [comment, marked as answer]
The sync batch indicates that there are sessions that should be uploaded to the cloud.
After saving or editing the session, a batch will be displayed.
If auto sync is enabled, sync will start after a few seconds.

**Comment by rediffusion** [comment]
1. Automatically sync.
   1) There is no synchronization while saving the session.
   2) There’s nothing when you close your browser too.
   3) Only when the browser is opened does synchronization occur.
   4) The exclamation point does not disappear on the cloud icon. [Screenshot](https://i.imgur.com/2t5JQ0R.png).
3. Sync auto save sessions.

            How does this feature works❓Please show me examples...
